app/models/Preset.java

```diff
@@ -0,0 +1,44 @@
+package models;
+
+import com.avaje.ebean.Model;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import play.data.validation.Constraints;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+/**
+ * Created by Shane on 3-5-2016.
+ */
+@Entity
+public class Preset extends Model {
+
+    @Id
+    public Long id;
+
+    public String name;
+    public int camera;
```

Comment:
There hasn't yet been made a camera object, and as there is still some discussion on how the object will act, for now it is linked with an integer id. Also the camera object may only be active as a live object and thus not stored in the database, for this id linking is also better.